samples/apps/bookinfo/route-rule-ratings-test-v2.yaml

```diff
@@ -0,0 +1,17 @@
+type: route-rule
+spec:
+  name: ratings-test-v2
```

Comment:
Since the rule has no specific (test related) match criteria and just sends 100% of traffic to v2, I think it might be better to just name this rule `ratings-default` and use `istioctl replace` rather than creating a second rule with higher precedence. I guess it depends on how we integrate this with the tasks and story flow. If we plan to run it and then delete before moving on to the next task, then I guess it's fine as is.